ros/api/v1/hosts.py

```diff
@@ -27,17 +30,16 @@
 
 class IsROSConfiguredApi(Resource):
     def get(self):
-        ident = identity(request)['identity']
-        account_query = db.session.query(RhAccount.id).filter(
-            RhAccount.account == ident['account_number']).subquery()
-        system_count = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query)).count()
-        systems_with_suggestions = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query))\
-            .filter(System.number_of_recommendations > 0).count()
-        systems_waiting_for_data = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query))\
-            .filter(System.state == 'Waiting for data').count()
+        account_number = identity(request)['identity']['account_number']
+        system_query = default_queries(account_number)
+        query = (
+            db.session.query(System.id)
+            .filter(PerformanceProfile.system_id.in_(system_query.subquery()))
+            .distinct()
+        )
```

Comment:
@apuntamb  - Is this query working correctly on local?

Reply:
yes @kgaikwad